pkg/config/config.go

```diff
@@ -1539,7 +1541,16 @@ func setupFipsEndpoints(config Config) error {
 	os.Unsetenv("HTTPS_PROXY")
 
 	// HTTP for now, will soon be updated to HTTPS
-	protocol := "http://"
+	var protocol string
+	if config.GetBool("fips.https") {
+		protocol = "https://"
+		// top-level setting takes precedence
```

Comment:
nit: This may need to be more specific: `// When 'skip_ssl_validation' is disabled, fips.tls_verify' setting is ignored`. We may even want a warning if that happens.